src/main/java/ch/njol/skript/conditions/CondItemEnchantmentGlint.java

```diff
@@ -0,0 +1,83 @@
+package ch.njol.skript.conditions;
+
+import org.bukkit.event.Event;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+@Name("Item Has Enchantment Glint Override")
+@Description("Checks whether an item has the enchantment glint overridden, or is forced to glint or not.")
+@Examples({
+	"if the player's tool has the enchantment glint override",
+		"\tsend \"Your tool has the enchantment glint override.\" to player",
+	"",
+	"if {_item} is forced to glint:",
+		"\tsend \"This item is forced to glint.\" to player",
+	"else if {_item} is forced to not glint:",
+		"\tsend \"This item is forced to not glint.\" to player",
+	"else:",
+		"\tsend \"This item does not have any glint override.\" to player"
+})
+@RequiredPlugins("Spigot 1.20.5+")
+@Since("INSERT VERSION")
+public class CondItemEnchantmentGlint extends Condition {
+
+	static {
+		if (Skript.isRunningMinecraft(1, 20, 5))
```

Comment:
A method exists check would be better

Reply:
le bump

Reply:
I have the rest of the changes resolved, looks like I forgot to push 😬😬